lib/vulcan.js

```diff
@@ -391,6 +381,27 @@ Vulcan.prototype = {
         return this.inlineCss(docObj.doc, docObj.href);
       }.bind(this));
     }
+
+    if (this.stripComments) {
```

Comment:
There is a check at https://github.com/Polymer/vulcanize/pull/293/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R598, but I made it more explicit in 28a7812